open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/entities/glossaries/SubjectAreaGlossary.java

```diff
@@ -2,9 +2,18 @@
 /* Copyright Contributors to the ODPi Egeria project. */
 package org.odpi.openmetadata.accessservices.subjectarea.client.entities.glossaries;
 
+import org.odpi.openmetadata.accessservices.subjectarea.client.SubjectAreaClient;
 import org.odpi.openmetadata.accessservices.subjectarea.client.SubjectAreaEntityClient;
 import org.odpi.openmetadata.accessservices.subjectarea.properties.objects.glossary.Glossary;
 
+/**
+ * The SubjectAreaDefinition Open Metadata Access Service (OMAS) API for glossaries.
+ */
 public interface SubjectAreaGlossary {
+
+   /**
+    * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+    * when working with Glossary objects
```

Comment:
Glossary objects => Glossaries. If we can avoid using object that would be good as this would have to be another concept we would need to explain/define